examples/tasks/enaml/employee_view.enaml

```diff
@@ -30,7 +30,7 @@ enamldef EmployeeForm(Form):
         text = 'Password'
     Field:
         echo_mode << 'password' if not pw_cb.checked else 'normal'
-        text :: print 'Password:', text
+        text :: print('Password:'), text
```

Comment:
On python 2, if i remember correctly, `print 'Password:', text` will print the value of the variable `text` after the string `"Password:"`. So, the right fix here is something like `print("Password: {}".format(text))` or something of that sort.